### PR TITLE
Script that checks API/ABI compatibility between versions

### PR DESCRIPTION
**Pull request by idelpivnitskiy**
Motivation:

It's hard to track API/ABI changes and list everything correctly in
release notes. This script produces summary of all changes made between
versions.

Modifications:

- Add `./scripts/japicmp.sh`;

Result:

Scripts that summarizes all API/ABI changes.

### CODE REVIEW

File: scripts/japicmp.sh [hunk 1]
```diff
@@ -34,16 +37,21 @@ if [ -z "$GROUP_ID" ]; then
...
 GROUP_PATH=$(echo "$GROUP_ID" | tr '.' '/')
-BASEPATH=$HOME/.m2/repository/$GROUP_PATH/
+BASEPATH=$MVN_REPO/$GROUP_PATH/
 
-# All servicetalk modules except: servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal:
+# All servicetalk modules except:
+# servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal
 ARTIFACTS="$(ls -d -- */ | grep 'servicetalk-' | sed 's/.$//' | \
-  grep -v 'benchmark' | grep -v 'bom' | grep -v 'examples' | grep -v 'gradle-plugin-internal')"
+  grep -v 'benchmark' | grep -v 'bom' | grep -v 'examples' | grep -v 'gradle-plugin-internal' | grep 'grpc')"
```
Comment (bondolo) [inline scripts/japicmp.sh:45]: only grpc?

File: scripts/japicmp.sh [hunk 2]
```diff
@@ -34,16 +37,21 @@ if [ -z "$GROUP_ID" ]; then
   GROUP_ID=io.servicetalk
 fi
 GROUP_PATH=$(echo "$GROUP_ID" | tr '.' '/')
-BASEPATH=$HOME/.m2/repository/$GROUP_PATH/
+BASEPATH=$MVN_REPO/$GROUP_PATH/
 
-# All servicetalk modules except: servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal:
+# All servicetalk modules except:
+# servicetalk-benchmarks, servicetalk-bom, servicetalk-examples, servicetalk-gradle-plugin-internal
 ARTIFACTS="$(ls -d -- */ | grep 'servicetalk-' | sed 's/.$//' | \
```
Comment (bondolo) [inline scripts/japicmp.sh:44]: I usually prefer `find` vs `ls` if the names may contain spaces.

File: scripts/japicmp.sh (hunk 1 above)
Comment (idelpivnitskiy) [reply scripts/japicmp.sh]: leftover

File: scripts/japicmp.sh (hunk 2 above)
Comment (idelpivnitskiy) [reply scripts/japicmp.sh]: It's known that the names of modules don't contain spaces

**Review by bondolo — APPROVED** [review]

**Pull request merged**